### PR TITLE
Only send sigverify to GPU if batch size is >64

### PR DESCRIPTION
**Pull request by sakridge**
Seems to be a decent crossover point for Xeon E5-2620 v4 8c,16t vs. nvidia 1080ti

### CODE REVIEW

File: src/sigverify.rs [hunk 1]
```diff
@@ -134,6 +137,9 @@ pub fn init() {
 pub fn ed25519_verify(batches: &Vec<SharedPackets>) -> Vec<Vec<u8>> {
     use packet::PACKET_DATA_SIZE;
     let count = batch_size(batches);
+    if count < 64 {
```
Comment (mvines) [inline src/sigverify.rs]: worthwhile adding a quick comment to justify`64`?

**Comment by sakridge** [comment]
Hopefully fixes https://github.com/solana-labs/solana/issues/817

**Comment by garious** [comment]
Why not make the GPU version work for <64 instead? We don't care about the speed of small batches.

**Comment by sakridge** [comment]
@garious it does work.. it's just slow. Why do you want to use the GPU to verify 1 signature which will waste all the lanes of the SIMD and the CPU has much better single-core performance.

**Comment by sakridge** [comment]
I guess you mean the startup issue... there are multiple potential fixes but this is one and it will improve latency for small batch requests.

**Comment by garious** [comment]
Well you look at the cost-benefit here: if large batches are sufficiently fast, then small batches must be sufficiently fast. Therefore optimizing small batches makes them more sufficiently fast? What's the tangible benefit?  As for the cost, it's very real. Nearly every unit test would take the small batch path. If there's a bug in the large batch path, it'd go unnoticed until stress testing, which is a harder environment to debug from.

**Pull request merged**